clients/web/src/templates/widgets/itemList.jade

```diff
@@ -2,16 +2,16 @@ ul.g-item-list
   each item in items
     li.g-item-list-entry
       if checkboxes
-        input.g-list-checkbox(type="checkbox", g-item-cid="#{item.cid}")
-      a.g-item-list-link.g-right-border(g-item-cid="#{item.cid}")
+        input.g-list-checkbox(type="checkbox", g-item-cid=item.cid)
+      a.g-item-list-link.g-right-border(g-item-cid=item.cid)
         i.icon-doc-text-inv
-        | #{item.get('name')}
```

Comment:
This, and the other unrelated changes, should be part of a separate commit, or even a separate PR.